pkg/scheduler/uthelper/helper.go

```diff
@@ -180,6 +185,14 @@ func (test *TestCommonStruct) CheckBind(caseIndex int) error {
 		}
 	}
 
+	if test.MinimalBindCheck {
+		return nil
```

Comment:
This return should put after the next select case, in case that if ExpectBindsNum is 0, but actually there are binds.

Reply:
done